datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -21,8 +21,8 @@ gearman==2.0.2; sys_platform != 'win32' and python_version < '3.0'
 httplib2==0.18.1
 in-toto==0.4.2
 ipaddress==1.0.22; python_version < '3.0'
-jaydebeapi==1.1.1
-jpype1==0.7.0
+jaydebeapi==1.2.3
+jpype1==0.7.1 ; python_version <= '2.7'  # 0.7.5 it's not py2 compatible but this specific version breaks in py3
```

Comment:
on py3 jaydebeapi will bring latest jpype: https://github.com/baztian/jaydebeapi/blob/master/setup.py#L23

Reply:
Shall we raise an issue on `jaydebeapi` repo about that ?